src/Octoshift/Services/OctoLogger.cs

```diff
@@ -33,7 +33,8 @@ public class OctoLogger
 
     private readonly List<string> _redactionPatterns = new()
     {
-        "\\b(?<=token=)(.+?)\\b"
+        "\\b(?<=token=)(.+?)\\b",
+        "\\b(?<=X-Amz-Credential=)(.+?)\\b",
```

Comment:
It'd be probably a good idea to also add case insensitivity to the regex match:
```C#
Regex.Replace(result, redactionPattern, "***", RegexOptions.IgnoreCase)
```

Reply:
Yes - that makes sense! I'll do that tomorrow and then I'll ping you for a fresh review.

Reply:
Addressed in b7bdc44